microovn/client/client.go

```diff
@@ -264,3 +265,25 @@ func DeleteConfig(ctx context.Context, c *client.Client, key string) (types.Dele
 
 	return responseData, err
 }
+
+// GetClientFromName gets a microcluster client by its name, returning an error
+// if one is not found
+func GetClientFromName(c *client.Client, m *microcluster.MicroCluster, nodeName string) (*client.Client, error) {
+	clusterMembers, err := c.GetClusterMembers(context.Background())
+	if err != nil {
+		return nil, err
+	}
+
+	for _, clusterMember := range clusterMembers {
+		if clusterMember.Name == nodeName {
+			cli, err := m.RemoteClient(clusterMember.Address.String())
+			if err != nil {
+				return nil, err
+			}
+
+			return cli, nil
+		}
+	}
+
+	return nil, errors.New("no such node")
```

Comment:
This error message could use more context. It could contain the name of the node that wasn't found.